### PR TITLE
chore: update 'listview' demo for nativescript-ui-listview v6.2.0

### PR DESCRIPTION
**Pull request by nsplugins**

### CODE REVIEW

File: listview/tsfmt.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (zbranzov) [inline listview/tsfmt.json]: This file should not be committed

File: listview/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
```
Comment (zbranzov) [inline listview/.editorconfig]: Should we commit this file ?

File: listview/tsfmt.json (hunk 1 above)
Comment (VladimirAmiorkov) [reply listview/tsfmt.json]: This is an autogenerated file that sets a "common" style when editing the demo-angular via VS Code. I think we should leave those files committed to enforce the same code style and formatting.

File: listview/.editorconfig (hunk 2 above)
Comment (VladimirAmiorkov) [reply listview/.editorconfig]: This is an autogenerated file that sets a "common" style when editing the demo-angular via VS Code. I think we should leave those files committed to enforce the same code style and formatting.

**Pull request closed without merging**